lib/cisco_node_utils/bgp.rb

```diff
@@ -360,6 +360,9 @@ def default_enforce_first_as
     # Nvgen as True With optional 'size <size>
     def event_history_cli
       match = config_get('bgp', 'event_history_cli', @get_args)
+      if match == default_event_history_cli
```

Comment:
This method already returns the default value at the end. Can you explain why this is needed here?